megamek/src/megamek/client/ui/swing/boardview/BoardView.java

```diff
@@ -5453,22 +5452,30 @@ public String getHexTooltip(MouseEvent e) {
         if (!game.getBoard().contains(mcoords)) {
             return null;
         }
+
         Hex mhex = game.getBoard().getHex(mcoords);
 
-        StringBuffer txt = new StringBuffer(HTML_BEGIN);
-        txt.append("<div WIDTH=500>");
+        String b = "";
+        String f = "";
+        String t = "";
+        String result = "";
+        String i = "";
```

Comment:
Please use more descriptive variable names. This provides no information on what this variable does.